packages/lumx-react/src/components/alert-dialog/AlertDialog.tsx

```diff
@@ -0,0 +1,167 @@
+    title?: string;
+    className?: string;
+    /** The props to the confirm button */
+    confirmProps: {
+        onClick(): void;
+        label: string;
+    };
+    /**
+     * The props to the cancel button.
+     * Will not render a cancel button if undefined.
+     */
+    cancelProps?: {
+        onClick(): void;
+        label: string;
+    };
+}
+
+/**
+ * Associative map from message kind to color and icon.
+ */
+const CONFIG = {
+    [Kind.error]: { color: ColorPalette.red, icon: mdiAlert },
+    [Kind.info]: { color: ColorPalette.blue, icon: mdiInformation },
+    [Kind.success]: { color: ColorPalette.green, icon: mdiCheckCircle },
+    [Kind.warning]: { color: ColorPalette.yellow, icon: mdiAlertCircle },
+};
+
+/**
+ * Component display name.
+ */
+const COMPONENT_NAME = 'AlertDialog';
+
+/**
+ * Component default class name and class prefix.
+ */
+const CLASSNAME = getRootClassName(COMPONENT_NAME);
+
+/**
+ * Component default props.
+ */
+const DEFAULT_PROPS: Partial<DialogProps> = {
+    size: Size.tiny,
+    kind: Kind.info,
+};
+
+/**
+ * AlertDialog component.
+ *
+ * An alert dialog is a modal dialog that interrupts the user's workflow to
+ * communicate an important message and acquire a response.
+ *
+ * It should not have a complexe content.
+ * Children of this component should only be strings, paragraphs or links.
+ */
+export const AlertDialog: Comp<AlertDialogProps, HTMLDivElement> = forwardRef(
+    (
+        { id, title, className, cancelProps, confirmProps, kind, size, dialogProps = {}, children, ...forwardedProps },
+        ref,
+    ) => {
+        const cancelButtonRef = React.useRef(null);
+        const confirmationButtonRef = React.useRef(null);
+        const { color, icon } = CONFIG[kind as Kind] || {};
+
+        // Define a unique ID to target title and description for aria attributes.
+        const uniqueId = id || uid();
+        const titleId = `${uniqueId}-title`;
+        const descriptionId = `${uniqueId}-description`;
+
+        // If content is a string, set in a paragraph.
+        const DescriptionElement = typeof children === 'string' ? 'p' : 'div';
+
+        return (
+            <Dialog
+                ref={ref}
+                focusElement={cancelProps ? cancelButtonRef : confirmationButtonRef}
+                size={size}
+                dialogProps={{
+                    id: uniqueId,
+                    role: 'alertdialog',
+                    'aria-labelledby': titleId,
+                    'aria-describedby': descriptionId,
+                    ...dialogProps,
+                }}
+                className={classNames(
+                    className,
+                    handleBasicClasses({
+                        kind,
+                        prefix: CLASSNAME,
+                    }),
+                )}
+                {...forwardedProps}
+            >
+                <header>
+                    <FlexBox
```

Comment:
I think @matmkian asked to use <Toolbar> on dialog header and footer. It implements good margin by default

Reply:
Ah, I didn't know that! 
Will update.

Reply:
It doesn't seem to work :( 
Is there something else to do to make it work ?

```javascript
            header={
                <FlexBox
                    orientation={Orientation.horizontal}
                    hAlign={Alignment.center}
                >
                    {icon && <Icon className="lumx-spacing-margin-right-big" icon={icon} size={Size.s} color={color} />}
                    <h2 id={titleId} className="lumx-typography-title">
                        {title}
                    </h2>
                </FlexBox>
            }
```
![image](https://user-images.githubusercontent.com/7147342/106027150-e6fe0700-60ca-11eb-8cfd-7d34057f3b95.png)

Reply:
I checked the css and the html, I don't see anything that would add a margin to the header or footer :sweat_smile:

Reply:
Ah, you meant the `Toolbar` component. Got it. 
Added ;)